setup.py

```diff
@@ -0,0 +1,40 @@
+from setuptools import setup
+
+with open('README.md', 'r') as fh:
+    long_description = fh.read()
+
+setup(
+    name='borealis-fireworks',
+    version='0.1.0',
+    packages=['borealis', 'borealis.util'],
+    url='https://github.com/CovertLab/borealis',
+    license='MIT',
+    author='Jerry Morrison',
+    author_email='j.erry.morrison@gmail.com',
+    description='Run FireWorks workflows in Google Cloud',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    install_requires=[
```

Comment:
Ah, this is a better approach than simply reading in the `requirements.txt`, we should probably do this in our other packages as well to play more nicely with other environments.

Reply:
Maybe even leave the `requirements.txt` file out of the build?

Reply:
Yeah it doesn't need to be in the build if we are not using it. No one uses it on the client side, that all relies on `install_requires` in setup.py, like here.